RocketChat/Rocket.Chat.ReactNative
# fix: iOS upload a file with gif and heic format

## Proposed changes


The fix on the native side is here: https://github.com/RocketChat/react-native-image-crop-picker/pull/6

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4178

## How to test or reproduce

- Upload a GIF from Choose from library

- Also, select a HEIC image and check if the image is uploaded in JPEG format and whether the image is displayed properly in other devices.

## Screenshots

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/963d14bc-e4cc-412d-93bd-3bbf2c1ad0a5


## Types of changes




## Checklist



## Further comments

[TC-398]

[TC-398]: https://rocketchat.atlassian.net/browse/TC-398?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
